README.md

```diff
@@ -31,35 +31,58 @@ The ```main``` branch of this repository contains samples configured for the lat
 
 The *ArcGIS Runtime SDK Samples app* has a *Target SDK* version of *13.0*, meaning that it can run on devices with *iOS 13.0* or newer.
 
-## Building samples using installed SDK
+## Building Samples Using Swift Package Manager
 
 1. **Fork** and then **clone** the repository
-    > Make sure to use the "recursive" option to ensure you get the **ArcGIS Runtime Toolkit** submodule
-    >
-    >`git clone --recursive [URL to Git repo]`
-    >
-    > If you've already cloned the repo without the submodule, you can load the submodule using 
-    >
-    >`git submodule update --init`
+1. **Run** the `arcgis-ios-sdk-samples` app target
+
+> The project is configured to install the ArcGIS Runtime SDK for iOS and ArcGIS Runtime Toolkit with Swift Package Manager.
+> To add the Swift Packages to your project, follow the steps below
+
+* Open your project in Xcode
+* Go to File > Swift Packages > Add Package Dependency option
+* Enter `https://github.com/Esri/arcgis-runtime-toolkit-ios` as the package repository URL
+* Choose version 100.11.0 or a later version. Click Next. Only version 100.11.0 or newer supports Swift Package Manager.
+
+> Note: The Toolkit Swift Package adds the ArcGIS Runtime SDK Swift Package as a dependency, so there is no need to add both separately.
+> 
+> New to Swift Package Manager? Visit [here](https://swift.org/package-manager/)
+
```

Comment:
Curious why you have these instructions here. Isn't the project already configured with the swift packages that are needed? Or do users have to explicitly add the packages as you describe here

Reply:
I think they are redundant, but willing to know others thoughts! We can remove these lines...

For SPM setup it is as simple as download and run the project. But users may wonder how to configure SPM similar to our project, so I added these instructions.

Reply:
Maybe show how you added the SP by adding a link to the toolkit README where it explains how the Toolkit SP is added to a project using SPM.

Reply:
Added a link to the projected `main` branch URL for the toolkit root README.